gradle.properties

```diff
@@ -4,8 +4,8 @@
 pluginGroup = org.nixos.idea
 pluginName = NixIDEA
 pluginVersion = 0.4.0.7
-pluginSinceBuild = 213
-pluginUntilBuild = 223.*
+pluginSinceBuild = 221
+pluginUntilBuild = 231.*
 
 platformType = IC
-platformVersion = 2021.3.3
+platformVersion = 2022.1
```

Comment:
You could have used version `2022.1.4` as platform version, but I guess it doesn't really matter.

Reply:
I was actually trying to figure out exactly this but did not manage to find the info, so just proceeded with what's in the docs https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html#intellij-platform-based-products-of-recent-ide-versions

Reply:
I usually use the latest patch releases, which you can find at the Downloads page under [Other Versions](https://www.jetbrains.com/idea/download/other.html). Anyway, I think it is not very important.